tests/TestCase.php

```diff
@@ -1,41 +1,56 @@
 <?php
 
+declare(strict_types=1);
+
```

Comment:
I'll have someone go through all the packages and add this to every php file.

Reply:
Its not so valuable that its worth dedicated effort IMO, its just habit for me to always add it when I find myself in a file without it.